AssertionError: Trying to copy from a source that does not exist

When executing the generator with the following choices, I'm getting an AssertionError:

Project name: whatever
Description: whatever
Type: REST, Realtime via Socket.io
CORS config: Enabled for all domains
Database: I will choose my own
Authentication: local

The exact error is: `AssertionError: Trying to copy from a source that does not exist: /usr/local/lib/node_modules/generator-feathers/service/templates/database-service.js`.

--- a/app/index.js
+++ b/app/index.js
@@ -93,6 +93,7 @@ module.exports = generators.Base.extend({
         choices: [
           {
             name: 'I will choose my own',
+            value: 'generic',
             checked: true
           },
           {

--- a/service/index.js
+++ b/service/index.js
@@ -153,6 +153,9 @@ module.exports = generators.Base.extend({
         case 'nedb':
           this.props.type = 'nedb';
           break;
+        default:
+          this.props.type = 'generic';
+          break;
       }
     }
 

